Gems/RosRobotSample/Assets/ROSbot_velodyne.prefab

```diff
@@ -386,6 +371,20 @@
                                 "IsStatic": false,
                                 "InterpolatePosition": 0,
                                 "InterpolateRotation": 0
+                            },
+                            "ROS2FrameEditorComponent": {
+                                "$type": "ROS2FrameEditorComponent",
+                                "Id": 88600217186346884,
+                                "ROS2FrameConfiguration": {
+                                    "Id": 0,
+                                    "Namespace Configuration": {
+                                        "Namespace Strategy": 0,
+                                        "Namespace": ""
+                                    },
+                                    "Frame Name": "velodyne_puck_link",
+                                    "Joint Name": "",
```

Comment:
Out of curiosity: why did you name the joint in _slamtec_ prefab, but not in this one? Is there any different?

Reply:
I'm not sure if I understand what are you referring to. I've checked the names and they haven't changed. Also this process was done using a script, all of it was automatic.